README.md

```diff
@@ -6,13 +6,50 @@ and later on implementing features or enhancements in Godot Engine,
 this repository is used to centralize all the proposals in one place
 apart from bug reports.
 
-Proposals are made by opening issues in this repository, while will
+Proposals are made by opening issues in this repository, which will
 then be discussed with fellow Godot users and contributors. Once a
 proposal is considered good to implement, it will be turned in a formal
 Godot Improvement Proposal (GIP) stored in text format in this
 repository.
 
-**Note:** Work in progress! Bear with us while we discuss and define
-the exact workflow that we want to use. Please refrain from opening
-proposals on this repository just yet, unless you are among the core
-contributors and discuss it beforehand.
+#### Rules for Submitting a Proposal
+
+1. Only proposals that properly fill out the template will be considered. If
+the template is not filled out or is filled out improperly it will be closed.
+
+2. All proposals must be linked to a substantive use-case. In justifying your
+proposal it is not enough to say it would be "nice" or "helpful". Use the
+template to show how Godot is not currently meeting your needs and then
+explain how your proposal will meet a particular need. 
+
+3. Proposals have similar requirements to Pull Requests (PRs) in the main repository.
+The graph in [this article](https://godotengine.org/article/will-your-contribution-be-merged-heres-how-tell)
+outlines what core developers consider when evaluating your PR/proposal.
+
+4. Other users must express interest in your proposal for it to be considered.
+Godot is community driven, if no other users are interested in your proposal,
+it may be closed. It is up to you to draw interest in your proposed feature.
+Start by reaching out on the community channels (Reddit, Discord, IRC, etc.
+see the [Community Channels](http://docs.godotengine.org/en/stable/community/channels.html) doc),
+then create your proposal once you have gained some interest.
+
+5. Feel free to make a PR before a proposal. If it is a large change a core
+developer may require a proposal be made and discussed before merging.
+
+6. If you or another user is capable of making a PR, include that fact in 
+the issue or in a subsequent comment so that a core contributor can 
```

Comment:
```suggestion
the issue or in a subsequent comment, so that a core contributor can 
```

Reply:
A comma shouldn't be added here. You only place a comma before "so" when it is being used as a coordinating conjunction.